stdlib/REPL/src/latex_symbols.jl

```diff
@@ -211,27 +228,24 @@ const latex_symbols = Dict(
     "\\euro" => "€",
 
 # 732 symbols generated from unicode.xml
-    "\\textexclamdown" => "¡",
+    "\\exclamdown" => "¡",
     "\\sterling" => "£",
     "\\yen" => "¥",
-    "\\textbrokenbar" => "¦",
+    "\\brokenbar" => "¦",
     "\\S" => "§",
-    "\\textasciidieresis" => "¨",
     "\\copyright" => "©",
-    "\\textordfeminine" => "ª",
+    "\\ordfeminine" => "ª",
     "\\neg" => "¬",
     "\\circledR" => "®",
-    "\\textasciimacron" => "¯",
```

Comment:
Is there a replacement for this?

Reply:
No.  But it wasn't clear to me that we needed this — this is the *non-combining* macron.  Who needs this?

Reply:
Isn't this the APL high minus?

Reply:
@ararslan, no, that would be [u+207b](https://www.fileformat.info/info/unicode/char/207b/index.htm), which we can already tab-complete via `\^-`.

A macron is not really intended to be used as a mathematical operator.

Reply:
I take it back, you are right: APL uses this for the "high minus", apparently.   I'll can add it back in as `\highminus` for the benefit of APL programmers.

Reply:
IIRC it's used in APL.jl, so perhaps worth keeping around, if only for that 😉

Reply:
I added it back as `\highminus`.